Project Styling Template (Members)

ganmany5288 opened the pull request:
I think this is the right place lol

Made a template for members, social medias, and the member's image that can be used across different pages in the website (courseup, vikelabs, ...)

ganmany5288 committed:
project template added for projects section

ganmany5288 committed:
front end CSS fixes

ganmany5288 committed:
Social Icon Update

CSwapped from using 'react-social-icon' library to 'react-icon' library

ganmany5288 committed:
Social Icon Update

For some reason there's no need for hovering styling to be made... tf is happening lol

ganmany5288 committed:
Part 1 Code cleanup

General code cleanup, removed unnecessary comments and codes and stylings

ganmany5288 committed:
Clean Up Part 2

Organized components and sectioned some into their own files located within the same folder

robchendev commented on src/templates/project/member-media.js:
```diff
@@ -0,0 +1,39 @@
+    )
+}
+
+const Media = (props) => {
+    return (
+        <MediaCheck media={props.name} />
+    )
+}
```
This seems redundant, why not just have `Media` return the entire function scope of `MediaCheck`?

ganmany5288 replied:
Didn't have that considered lmao

robchendev reviewed (commented):
LGTM, code style needs some touch-up as mentioned in the code review comments. Look over the review, fix some things to your own preference and upon your next push I'll merge.

ganmany5288 committed:
Code cleanup part tres

Made the following cleanups

1. destructured props within project-media and project-member for easier reading of props used
2. made other changes to project-media
3. Added new color lightPurple to theme and gatsby-ssr
4. Removed unnecessary imports from some files (won't be used)
5. Removed some comments from project
6. Removed unnecessary styling
7. Added sample embedded links to media icons
8. Other general cleanups

ganmany5288 committed:
Final Code cleanup

Stacked props

robchendev reviewed (approved):
Very nice work, I'll be making some code-style related changes and maybe some design changes in a later PR.
